public/locales/en/messages.po

```diff
@@ -0,0 +1,34 @@
+msgid ""
+msgstr ""
+"POT-Creation-Date: 2023-02-07 12:02+0100\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: @lingui/cli\n"
+"Language: en\n"
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"PO-Revision-Date: \n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"Plural-Forms: \n"
+
+#: src/containers/CartItem/CartItem.js:108
+msgid "Decrease quantity to {0}"
+msgstr ""
+
```

Comment:
This file should probably also be removed.
Maybe on `npx @noaignite/create-app`?

Reply:
So you mean this file does not need to exist for the app to work?